tests/MongoDB.Analyzer.Tests.Common.TestCases/Linq/LinqExpressionsBasicTestCases.cs

```diff
@@ -0,0 +1,46 @@
+﻿using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using System.Linq;
+
+namespace MongoDB.Analyzer.Tests.Common.TestCases.Linq
+{
+    public sealed class LinqExpressionsBasicTestCases : TestCasesBase
+    {
+        // MongoLinq2MQ
+        // aggregate([{ "$match" : { "Name" : "Bob", "Age" : { "$gt" : 16, "$lte" : 21 } } }])
```

Comment:
As discussed in Slack, these comments would be better expressed as custom attributes. I believe you already changed this in a later PR, but wanted to confirm.